### PR TITLE
docs - updates to RELEASE/ROLLBACK TO... SAVEPOINT sql ref pages

### PR DESCRIPTION
**Pull request by lisakowen**
bring SAVEPOINT, RELEASE SAVEPOINT, and ROLLBACK TO SAVEPOINT sql ref pages up to postgres 12 content.  didnt notice any greenplum-specific content on the pages.

doc review site links (behind vpn):
- SAVEPOINT (can navigate to others from here):  https://docs-staging.vmware.com/en/draft/VMware-Tanzu-Greenplum/sqlref-savepoint/greenplum-database/GUID-ref_guide-sql_commands-SAVEPOINT.html

### CODE REVIEW

File: gpdb-doc/markdown/ref_guide/sql_commands/ROLLBACK_TO_SAVEPOINT.html.md [hunk 1]
```diff
@@ -5,12 +5,12 @@ Rolls back the current transaction to a savepoint.
...
 
 ``` {#sql_command_synopsis}
-ROLLBACK [WORK | TRANSACTION] TO [SAVEPOINT] <savepoint_name>
+ROLLBACK [ WORK | TRANSACTION ] TO [ SAVEPOINT ] <savepoint_name>
 ```
 
 ## <a id="section3"></a>Description 
 
-This command will roll back all commands that were run after the savepoint was established. The savepoint remains valid and can be rolled back to again later, if needed.
+This command roll backs all commands that were run after the savepoint was established. The savepoint remains valid and can be rolled back to again later, if needed.
```
Comment (dyozie) [inline gpdb-doc/markdown/ref_guide/sql_commands/ROLLBACK_TO_SAVEPOINT.html.md:13]: roll backs -> rolls back

?

**Review by dyozie — APPROVED** [review]

**Comment by lisakowen** [comment]
@Tao-T - did you get a chance to review this PR?

**Review by Tao-T — APPROVED** [review]
LGTM

**Comment by lisakowen** [comment]
thank you for the quick review, @Tao-T !

**Pull request merged**